test/data/tasks.js

```diff
@@ -195,5 +195,18 @@ module.exports = query => query.insert([
       changedBy: user.id
     },
     status: 'autoresolved'
+  },
+  {
+    id: uuid(),
+    data: {
+      data: {
+        conditions: 'conditions update'
```

Comment:
If you give this task a `name` then you don't need the custom assertion code that you wrote.

Reply:
I did think that, but then I thought it was unrealistic to change the "name" on an update-conditions task, I figured name made sense on the others as they were all profiles, but I was obviously overthinking it and should just use name :D

Reply:
Yeah, the data doesn't matter for the tests, so it was just a way of identifying the tasks for asserting in a human-readable way (the alternative would have been listing uuids)